app/lib/rocketchat.js

```diff
@@ -240,6 +242,10 @@ const RocketChat = {
 
 			sdkConnect();
 
+			this.connectedListener = this.sdk.onStreamData('connecting', () => {
+				reduxStore.dispatch(connectRequest());
+			});
+
 			this.connectedListener = this.sdk.onStreamData('connected', () => {
 				reduxStore.dispatch(connectSuccess());
 			});
```

Comment:
Just confirming... is this **double**-assignment to `this.connectedListener` correct and intended?

Reply:
@nmagedman it's not. Can you open a PR? Thank you!

Reply:
yes.  opening PR now...